### PR TITLE
Updated all links in README for Foxy and CLI Argument Fix

### PR DESCRIPTION
**Pull request by selimxsuha**
All the links are updated in the README file to point to the correct API documentation and repository. 
Additionally, the CLI argument formatting was fixed to adapt with the ROS2 syntax where arguments are passed using the format "joy_config:=xbox360" instead of using square brackets "[joy_config:=xbox360]"

### CODE REVIEW

**Comment by selimxsuha** [comment]
+ DualShock4 Config for Teleoperation has been added.

File: create_description/launch/create_2.launch [hunk 1]
```diff
@@ -3,4 +3,6 @@
   <node name="robot_state_publisher" pkg="robot_state_publisher" exec="robot_state_publisher" output="screen">
     <param name="robot_description" type="str" value="$(command '$(find-pkg-prefix xacro)/bin/xacro $(find-pkg-share create_description)/urdf/create_2.urdf.xacro')" />
   </node>
+  <!--Bumper to Point Cloud-->
+  <node pkg="rviz_scripts" exec="bumperToPointCloud" name="bumperToPointCloud" output="screen" />
```
Comment (jacobperron) [inline create_description/launch/create_2.launch]: This creates a runtime dependency on `rviz_scripts`. I'd rather not introduce a dependency for visualization to this description package.
```suggestion
```

File: create_bringup/launch/create_2.launch [hunk 2]
```diff
@@ -10,4 +10,5 @@
 
   <!-- Robot description -->
   <include if="$(var desc)" file="$(find-pkg-share create_description)/launch/create_2.launch" />
+
```
Comment (jacobperron) [inline create_bringup/launch/create_2.launch]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -166,48 +166,48 @@ $ ros2 launch create_bringup create_2.launch config:=/abs/path/to/config.yaml de
...
+`set_ascii` | Sets the 4 digit LEDs. Accepts 1 to 4 bytes, each representing an ASCII character to be displayed from left to right | [std_msgs/msg/UInt8MultiArray][uint8multiarray]
+`dock` | Activates the demo docking behaviour. Robot enters _Passive_ mode meaning the user loses control (See [OI Spec][oi_spec]) | [std_msgs/msg/Empty][empty]
+`undock` | Switches robot to _Full_ mode giving control back to the user | [std_msgs/msg/Empty][empty]
+`define_song` | Define a song with up to 16 notes. Each note is described by a MIDI note number and a float32 duration in seconds. The longest duration is 255/64 seconds. You can define up to 4 songs (See [OI Spec][oi_spec]) | [create_msgs/msg/DefineSong][definesong_msg]
+`play_song` | Play a predefined song | [create_msgs/msg/PlaySong][playsong_msg]
 
 ## Commanding your Create
 
-You can move the robot around by sending [geometry_msgs/Twist][twist] messages to the topic `cmd_vel`:
+You can move the robot around by sending [geometry_msgs/msg/Twist Message][twist] messages to the topic `cmd_vel`:
```
Comment (jacobperron) [inline README.md]:
```suggestion
You can move the robot around by sending [geometry_msgs/msg/Twist][twist] messages to the topic `cmd_vel`:
```

File: README.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ _* Not verified. Anyone who is able to verify that this driver works or not is e
 #### Prerequisites
 
 * Internet connection
-* [ROS](https://index.ros.org/doc/ros2/Installation) _Foxy_
+* [ROS2](https://index.ros.org/doc/ros2/Installation) _Foxy_
```
Comment (jacobperron) [inline README.md]:
```suggestion
* [ROS 2](https://index.ros.org/doc/ros2/Installation) _Foxy_
```

File: README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # create_robot
 
-[ROS](http://ros.org) driver for iRobot [Create 1 and 2](http://www.irobot.com/About-iRobot/STEM/Create-2.aspx).
+[ROS2](https://docs.ros.org/en/foxy/) Foxy driver for iRobot [Create 1 and 2](http://www.irobot.com/About-iRobot/STEM/Create-2.aspx).
```
Comment (jacobperron) [inline README.md]:
```suggestion
[ROS 2](https://docs.ros.org/en/foxy/) driver for iRobot [Create 1 and 2](http://www.irobot.com/About-iRobot/STEM/Create-2.aspx).
```

**Review by jacobperron — COMMENTED** [review]
Thanks for the updates! I've just made a couple small changes.

File: README.md (hunk 3 above)
Comment (jacobperron) [reply README.md:210]: Remove extra text

**Pull request merged**